models/sessions/scratch/snowplow_mobile_sessions_this_run.sql

```diff
@@ -20,7 +20,7 @@ select
 
   sa.start_tstamp,
   sa.end_tstamp,
-  {{ dbt_utils.current_timestamp() }} as model_tstamp,
+  {{ current_timestamp() }} as model_tstamp,
```

Comment:
Why is this `current_timestamp_in_utc` in web but just `current_timestamp` in mobile?

Reply:
Not sure but let's make them consistent across the board. Will change to `current_timestamp_in_utc`